srcpkgs/eclipse/template

```diff
@@ -14,12 +14,11 @@ homepage="https://eclipse.org"
 make_dirs="/usr/lib/eclipse/dropins 0770 root users"
 _patch=R
 _edition=java
-_mirror="http://ftp-stud.fht-esslingen.de/pub/Mirrors/eclipse"
+_mirror="http://mirror.umd.edu/eclipse"
```

Comment:
these variables aren't used anywhere but the url, they should just get merged into it

and https on the mirror